changelogs/unreleased/7528-add-link-to-discovery-resource.yml

```diff
@@ -0,0 +1,6 @@
+description: Add link to the resource responsible for the discovery to the discovered resource view.
```

Comment:
What do you mean with "the discovered resource view". I don't think this has a lot of meaning to an end-user.

Reply:
I meant the Discovered Resource page, but this is inaccurate, as the displaying of this link itself is not yet implemented on the frontend side. Changed this to:
```
  Update the endpoints returning discovered resource (`GET /api/v2/discovered/<discovered_resource_id>` and
  `GET /api/v2/discovered`) to include a link to the resource responsible for the discovery.
```